README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
@rprieto @nguyenchr I didn't get it! Shall I go with the `s.string({min, max})` and update for the rest ? E.g. `s.array(min, max, of)` to `s.array({min, max, of})`?